backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxRuleApi.scala

```diff
@@ -109,6 +109,7 @@ object VeloxRuleApi {
     injector.injectFinal(c => RemoveGlutenTableCacheColumnarToRow(c.session))
     injector.injectFinal(c => GlutenFallbackReporter(c.glutenConf, c.session))
     injector.injectFinal(_ => RemoveFallbackTagRule())
+    injector.injectFinal(c => GlutenAutoAdjustStageResourceProfile(c.glutenConf, c.session))
```

Comment:
Can we move the rule between `RemoveGlutenTableCacheColumnarToRow` and `GlutenFallbackReporter`? Thanks.